src/Functions/FunctionsBitmap.cpp

```diff
@@ -24,13 +24,207 @@ REGISTER_FUNCTION(Bitmap)
+        {"bitmap2", "Second bitmap object. [`AggregateFunction(groupBitmap, T)`](/sql-reference/data-types/aggregatefunction)."}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_bitmapXor = "Returns a bitmap containing elements present in either input bitmap, but not in both. [`AggregateFunction(groupBitmap, T)`](/sql-reference/data-types/aggregatefunction)";
+    FunctionDocumentation::Examples examples_bitmapXor = {{"Usage example", "SELECT bitmapToArray(bitmapXor(bitmapBuild([1,2,3]), bitmapBuild([3,4,5]))) AS res;",
+        R"(
+┌─res───────┐
+│ [1,2,4,5] │
+└───────────┘
+    )"}
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_bitmapXor = {20, 1};
+    FunctionDocumentation::Category category_bitmapXor = FunctionDocumentation::Category::Bitmap;
+    FunctionDocumentation documentation_bitmapXor = {
+        description_bitmapXor,
+        syntax_bitmapXor,
+        arguments_bitmapXor,
+        returned_value_bitmapXor,
+        examples_bitmapXor,
+        introduced_in_bitmapXor,
+        category_bitmapXor
+    };
+
+    factory.registerFunction<FunctionBitmapXor>(documentation_bitmapXor);
+
+    // Documentation for bitmapAndnot
+    FunctionDocumentation::Description description_bitmapAndnot = "Computes the logical conjunction of two bitmaps and negates the result (AND-NOT).";
+    FunctionDocumentation::Syntax syntax_bitmapAndnot = "bitmapAndnot(bitmap1, bitmap2)";
+    FunctionDocumentation::Arguments arguments_bitmapAndnot = {
+        {"bitmap1", "First bitmap object. [`AggregateFunction(groupBitmap, T)`](/sql-reference/data-types/aggregatefunction)."},
+        {"bitmap2", "Second bitmap object. [`AggregateFunction(groupBitmap, T)`](/sql-reference/data-types/aggregatefunction)."}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_bitmapAndnot = "Returns a bitmap containing elements present in the first bitmap but not in the second. [`AggregateFunction(groupBitmap, T)`](/sql-reference/data-types/aggregatefunction)";
+    FunctionDocumentation::Examples examples_bitmapAndnot = {{"Usage example", "SELECT bitmapToArray(bitmapAndnot(bitmapBuild([1,2,3]), bitmapBuild([3,4,5]))) AS res;",
+        R"(
+┌─res───┐
+│ [1,2] │
+└───────┘
+    )"}
+    };
+    FunctionDocumentation::IntroducedIn introduced_in_bitmapAndnot = {20, 1};
+    FunctionDocumentation::Category category_bitmapAndnot = FunctionDocumentation::Category::Bitmap;
+    FunctionDocumentation documentation_bitmapAndnot = {
+        description_bitmapAndnot,
+        syntax_bitmapAndnot,
+        arguments_bitmapAndnot,
+        returned_value_bitmapAndnot,
+        examples_bitmapAndnot,
+        introduced_in_bitmapAndnot,
+        category_bitmapAndnot
+    };
+
+    factory.registerFunction<FunctionBitmapAndnot>(documentation_bitmapAndnot);
+
+    // Documentation for bitmapHasAll
+    FunctionDocumentation::Description description_bitmapHasAll = "Checks if the first bitmap contains all elements of the second bitmap.";
```

Comment:
Lets replace `elements` everywhere by `bits`, my feeling is that such terminology fits better to bitmaps.